### PR TITLE
[Enhancement] Separate JNI logs from be.out

### PR DESCRIPTION
**Pull request by miomiocat**
- separate JNI logs from be.out
- rolling out config 
- provide log4j-1 compatibility using [log4j 1.2 bridge](https://logging.apache.org/log4j/2.x/log4j-1.2-api.html)

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Review by Smith-Cruise — APPROVED** [review]

File: conf/log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
+status = info
```
Comment (stephen-shelby) [inline conf/log4j2.properties:1]: error?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: conf/log4j2.properties (hunk 1 above)
Comment (miomiocat) [reply conf/log4j2.properties]: done

**Review by silverbullet233 — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by dirtysalt — APPROVED** [review]

**Comment by miomiocat** [comment]
@mergify backport branch-3.1

**Pull request merged**